apps/dotcom/sync-worker/src/UserDataSyncer.ts

```diff
@@ -126,15 +127,16 @@ export class UserDataSyncer {
 	private queue = new ExecutionQueue()
 
 	async reboot(delay = true) {
-		// TODO: set up analytics and alerts for this
 		this.debug('rebooting')
+		this.logEvent({ type: 'reboot', id: this.userId })
```

Comment:
We combine user durable object and user data syncer events under `user_durable_object` name. We could also separate them, but this felt good enough for now since these two act in tandem.